.github/workflows/post-merge.yml

```
@@ -0,0 +1,20 @@
+name: Post merge
+
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    uses: ./.github/workflows/run-tests.yml
+
+  release:
+    needs: test
```

Awww just noticed, the test job is `tests` not test
```suggestion
    needs: tests
```

---

Isn't it "test"? https://github.com/alphagov/pay-stream-s3-sqs/pull/89/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R11

---

It's the job in this file above on line 12

---

Fixed